packages/calendar/__tests__/calendar.spec.ts

```diff
@@ -61,29 +72,4 @@ describe('Calendar.vue', () => {
     expect(/2019.*May/.test((titleEl.element as HTMLElement).innerHTML)).toBeTruthy()
     expect(cell.classList.contains('is-selected')).toBeTruthy()
   })
-
```

Comment:
Any reason we should delete this test?

Reply:
firstDayOfWeek prop is no use in calendar code

Reply:
Although it's not used in the code, firstDayOfWeek is still a valid setting configed by dayjs.locale() in the global scope. 

The test is still needed.